test/integration/isolated_mode_test.js

```diff
@@ -0,0 +1,46 @@
+suite('Shutdown on idle', function() {
+  var co = require('co');
```

Comment:
for new tests that we write, do we want to still use co or follow babel/es6 conventions?